Formula/retroshare-service.rb

```diff
@@ -0,0 +1,52 @@
+class RetroshareService < Formula
+  desc "RetroShare friend to friend communication headless service"
+  homepage "https://retroshare.cc/"
+  url "https://github.com/RetroShare/RetroShare/releases/download/v0.6.6/RetroShare-v0.6.6-497-g2b4e57133-homebrew.tar.gz"
+  version "0.6.6-497-g2b4e57133-homebrew"
+  sha256 "69d9df59ea2566692d8c7cc8015a9ab100ed2578282bf7ccf4899e4f331a57fe"
+  license "AGPL-3.0-only"
+  head "https://github.com/RetroShare/RetroShare.git", branch: "master"
+
+  depends_on "cmake" => :build
+  depends_on "doxygen" => :build
+  depends_on "python@3.9" => :build
```

Comment:
does it work with 3.10?

Reply:
It works with any python 3 version, is there a way to express that, putting just python without specifying a version seems to break the CI tests

Reply:
In that case it should depend on the latest version (3.10)

Reply:
should I squash all commits together at the end?